tests/src/Functional/SmartLoaderDependenciesTest.php

```diff
@@ -48,7 +48,7 @@ public function testLibraryLoading(string $module, string $url, $setup_method =
     $this->container->get('kernel')->invalidateContainer();
 
     $this->drupalGet($url);
-    $this->assertSession()->responseContains('<script src="https://europa.eu/webtools/load.js');
+    $this->assertSession()->responseContains('<script src="https://webtools.europa.eu/load.js');
```

Comment:
Unless the `defer` attribute is added later in the code logic, we would recommend to add the `defer` attribute.
The Webtools Smartloader is usually loaded with `defer`.
See 
* https://webgate.ec.europa.eu/fpfis/wikis/display/webtools/Important+guidelines+for+developers+and+implementers
* https://webgate.ec.europa.eu/fpfis/wikis/display/webtools/Widgets